README.md

```diff
@@ -64,6 +64,27 @@ Optionally for MIDI support: TiMidity++ or FluidSynth and a soundfont.
 4. In game properties window select "Force the use of a specific Steam Play
    compatibility tool" and select "DOSBox (native)".
 
+
+## Installation (from source)
+
+0. Install extra prerequisites.
+
+#### Arch, Manjaro
+
+    $ sudo pacman -S git shellcheck python-pylint python-pyenchant hunspell-en_GB
+
+1. Close Steam.
+2. Clone the repository and install the script to user directory:
+
+       $ git clone https://github.com/dreamer/steam-dos.git
+       $ cd steam-dos
+       $ make user-install
+
+3. Start Steam.
+4. In game properties window select "Force the use of a specific Steam Play
+   compatibility tool" and select "DOSBox (native)".
```

Comment:
There's one quirk with `make user-install`, that might mislead some people and I would like to have it solved before putting this description to README document.

`user-install` puts the files into different directory (`steam-dos-dev`) than normal install (on purpose, so development version could be installed alongside stable version). However I never finished that transition and internal name of the tool is still `steam-dos`. Long story short: with these instructions user can't be sure which version is used if both (tarball and git) are installed at the same time.

Reply:
I actually think that the name of the tool should include a version it's based at, just like Proton (and even Proton uses "Beta" for some versions, you could add "Dev" there I guess), but it's for a different issue, especially as it's all in progress. Anything you want me to change in this PR for now till then?

Reply:
Implemented complete separate installation for `steam-dos` and `steam-dos-dev` and pushed it to master branch. The version installed from git shows up as "steam-dos (git)", but I am open to changing this display name.

Initially (until ~0.2, I think) I was experimenting with placing version number in there - same way Proton does, but Steam client does not refresh them automatically (needs Steam restart), while all other files used by the tool are used in newer version. Also, it sometimes resulted in games loosing compatibility tool altogether (and they show up in Steam interface as if they were natively Linux games, which is confusing as hell).

BTW, if `compatilitytool.vdf` and `toolmanifest.vdf` do not change, then it's not necessary to restart Steam after `make user-install`. But don't add this information here - it's already mentioned in `CONTRIBUTING.md`.

After you'll update display name to be selected after user-install, I'll merge your changes.